src/Neo/Cryptography/ECC/ECPoint.cs

```diff
@@ -23,9 +23,9 @@ namespace Neo.Cryptography.ECC
     /// </summary>
     public class ECPoint : IComparable<ECPoint>, IEquatable<ECPoint>, ISerializable
     {
-        internal ECFieldElement X, Y;
+        internal ECFieldElement? X, Y;
```

Comment:
It should not be null

Reply:
check the class, it has null assignment and null check.

Reply:
Maybe we can fix it, Why we want a point with null X and Y ? xD

Reply:
Infinity